### DISCUSSION TITLE
Gnome Overview vs Desktop View

### DISCUSSION THREAD

**Discussion by ghxstweb** [GradienceTeam/Gradience#837] [General]
Hi Gradience Devs. I just compiled gradience form source (using meson) in order to test the ability to theme the shell. Everything works well, however, I was wondering if there might be an option to theme the overview "view" along with the desktop view because it appears to have two different configurations which does look a bit odd. Youw can see at the top right that the icon colors for the system tray revert back to being white, after I had explicitly turned them to red.  Please let me know if you would like for me to clarify anything. Thanks in advance for your time!

These are two images below, which present the desktop view and overview view respectively:

![image](https://github.com/GradienceTeam/Gradience/assets/131561567/03b18353-0ff5-4f54-9b1e-a22b62c5feed)
![image](https://github.com/GradienceTeam/Gradience/assets/131561567/2db991eb-bfa6-4b78-8865-19d54b379b0f)

**Comment by Oren-Schmerling** [comment]
I've had the same issue. On a previous machine, a Gnome update fixed it, but I've gotten a new computer since then, which is on the latest version (Fedora 39 and GNOME 45.3) and the bug persists.

**Reply by ghxstweb** [reply]
any updates? i havent found a solution, lol

**Comment by Oren-Schmerling** [comment]
nothing on my end, but I got a nee laptop that’s black and white so I’m
just doing a white theme and giving up 🫠
